Shared/Logger.swift

```diff
@@ -19,8 +19,27 @@ public extension Logger {
     
     /// Logger used for things recorded on BraveRewards framework.
     static let rewardsLogger: RollingFileLogger = {
-        let logger = RollingFileLogger(filenameRoot: "rewards", logDirectoryPath: Logger.logFileDirectoryPath())
-        logger.setup(showFunctionName: false, showFileNames: false, showLineNumbers: false)
+        let logger = RollingFileLogger(filenameRoot: "rewards", logDirectoryPath: nil)
+        logger.identifier = "BraveRewards"
+        
+        if !AppConstants.BuildChannel.isRelease {
+            // Create a destination for the system console log (via NSLog)
+            let systemDestination = AppleSystemLogDestination(identifier: "com.brave.ios.logs")
+
+            systemDestination.outputLevel = .debug
+            systemDestination.showLogIdentifier = true
+            systemDestination.showLevel = true
+            systemDestination.showLevel = true
```

Comment:
This line was duped, not sure if its supposed to be something different?

Reply:
my mistake